configs/yolo/README.md

```diff
@@ -0,0 +1,25 @@
+# YOLOv3
+
+## Introduction
+```
+@misc{redmon2018yolov3,
+    title={YOLOv3: An Incremental Improvement},
+    author={Joseph Redmon and Ali Farhadi},
+    year={2018},
+    eprint={1804.02767},
+    archivePrefix={arXiv},
+    primaryClass={cs.CV}
+}
+```
+
+## Results and Models
+
+Test set: COCO val2017
```

Comment:
We may use a table to write down performance.

Reply:
It would be nice if you could provide the download link of the trained YOLOv3 model.

Reply:
Here is the problem. I don't know whether we should adopt the darknet-style training policy (~273 epochs on COCO, takes several days), or the detectron-style policy(1x, 2x, ...).

The trained model of 273 epochs is just there., provided by the previous author. I didn't train it, because I don't have many cards. However, I do have the ones with 1x and 2x training policy, which achieve ~3 mAP less than the 273-epoch one..